README.md

```diff
@@ -88,3 +88,12 @@ As well as uploading the messages, individuals, and production CSVs to Drive, th
  
 To make coded data available to the pipeline, use the `get.py` script in `CodaV2/data_tools` to export coded messages
 to `<data-root>/Coded Coda Files/<dataset>.json`
+
+
+## Development
+
+### Profiling
```

Comment:
Is this a sampling or a tracing profiler by default? Given that you need to interpret the results pretty differently it's worth being explicit

Reply:
Statistical, as mentioned below. Will bring that word forward by one sentence.